### PR TITLE
Chao/fix waiting for partuuid

### PR DESCRIPTION
**Pull request by AsuraZeng**

### CODE REVIEW

File: recipes-core/images/iot2050-image-example.bb [hunk 1]
```diff
@@ -60,6 +60,7 @@ IOT2050_DEBIAN_DEBUG_PACKAGES = " \
     net-tools \
     i2c-tools \
     sudo \
+    parted \
```
Comment (jan-kiszka) [inline recipes-core/images/iot2050-image-example.bb:63]: Belongs as dependency into the package that needs it. Fold it over there into the patch that adds the feature.

Comment (AsuraZeng) [reply recipes-core/images/iot2050-image-example.bb:63]: OK. Have added this package belong to the feature patch.

Comment (jan-kiszka) [reply recipes-core/images/iot2050-image-example.bb:63]: I'm not seeing this yet. It's still only in the image recipe, not in DEBIAN_DEPENDS of install-on-emmc.

File: recipes-core/install-on-emmc/install-on-emmc_1.0.bb [hunk 2]
```diff
@@ -11,7 +11,7 @@ inherit dpkg-raw
 DESCRIPTION = "This service provides the option to install on eMMC during first boot"
 
 DEPENDS = "regen-rootfs-uuid"
-DEBIAN_DEPENDS = "systemd, fdisk, util-linux, regen-rootfs-uuid"
+DEBIAN_DEPENDS = "systemd, fdisk, util-linux, regen-rootfs-uuid parted"
```
Comment (jan-kiszka) [inline recipes-core/install-on-emmc/install-on-emmc_1.0.bb:14]: Syntax error

**Review by jan-kiszka — CHANGES_REQUESTED** [review]

File: recipes-core/install-on-emmc/install-on-emmc_1.0.bb (hunk 2 above)
Comment (jan-kiszka) [reply recipes-core/install-on-emmc/install-on-emmc_1.0.bb:14]: (it's a subtle bug I make too often as well, but it is quickly found when running the build)

**Review by jan-kiszka — APPROVED** [review]

**Pull request merged**